RadarSDK/RadarLocationManager.m

```diff
@@ -437,10 +443,9 @@ - (void)removeBubbleGeofence {
 
 - (void)replaceSyncedGeofences:(NSArray<RadarGeofence *> *)geofences {
     [self removeSyncedGeofences];
+    [self removePendingNotifications];
 
-    BOOL tracking = [RadarSettings tracking];
-    RadarTrackingOptions *options = [Radar getTrackingOptions];
-    if (!tracking || !options.syncGeofences || !geofences) {
+    if (!geofences) {
```

Comment:
We go down this codepath regardless of whether or not we're tracking, because `nearbyGeofences` might come from a `trackOnce`